### PR TITLE
OCM-6468 | fix: change describe node pool output

### PR DESCRIPTION
**Pull request by oriAdler**
Add `Additional` to the security group IDs field.

### CODE REVIEW

File: cmd/describe/machinepool/cmd_test.go [hunk 1]
```diff
@@ -18,38 +18,38 @@ import (
...
+Instance type:                         m5.xlarge
+Labels:                                
+Taints:                                
+Availability zone:                     us-east-1a
+Subnet:                                
+Version:                               4.12.24
+Autorepair:                            No
+Tuning configs:                        
+Message:                               
+Additional security Group IDs:
```
Comment (ciaranRoche) [inline cmd/describe/machinepool/cmd_test.go:52]: NP, either capitalize `security` or make `Group` lower case :pray:

Comment (oriAdler) [reply cmd/describe/machinepool/cmd_test.go]: Changed, thanks.

**Comment by oriAdler** [comment]
@ciaranRoche I addressed the comments, can you rePTAL?

**Pull request merged**